packages/web/src/components/track/LockedStatusBadge.tsx

```diff
@@ -0,0 +1,20 @@
+import { IconLock, IconLockUnlocked } from '@audius/stems'
```

Comment:
I think I named it `iconUnlocked` in mobile (and I see it in stems as well), could be convinced that this name is better but let's decide on one.

Reply:
Cool agree!